streams/src/main/java/org/apache/kafka/streams/processor/StateStore.java

```diff
@@ -61,7 +61,7 @@
      * @throws IllegalStateException If store gets registered after initialized is already finished
      * @throws StreamsException if the store's change log does not contain the partition
      */
-    void init(ProcessorContext context, StateStore root);
+    void init(ProcessorContext<?, ?> context, StateStore root);
```

Comment:
Since we don't have types on StateStore, there's no reasonable restriction we can place on what calls the store can make to `ProcessorContext#forward`